src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,61 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { SortTypes } from '../../types/SortTypes';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type TodoFilterProps = {
+  query: string,
+  onChangeQuery: (value: string) => void;
+  selectedStatus: string,
+  onSelectStatus: (value: string) => void,
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+export const TodoFilter: React.FC<TodoFilterProps> = ({
+  query,
+  onChangeQuery,
+  onSelectStatus,
+}) => {
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select
+            data-cy="statusSelect"
+            onChange={(event) => onSelectStatus(event.target.value)}
+          >
+            {Object.entries(SortTypes).map(([key, value]) => (
+              <option
```

Comment:
When you map childrens in react the highest level component must have a key attribute with unique value
```suggestion
              <option
                  key={value}
```